[2022-06-27 13:33:16,980] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-06-14T00:00:00+00:00 [queued]>
[2022-06-27 13:33:17,030] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-06-14T00:00:00+00:00 [queued]>
[2022-06-27 13:33:17,033] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:33:17,034] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:33:17,035] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:33:17,066] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_A> on 2022-06-14 00:00:00+00:00
[2022-06-27 13:33:17,141] {standard_task_runner.py:52} INFO - Started process 58473 to run task
[2022-06-27 13:33:17,187] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_A', 'scheduled__2022-06-14T00:00:00+00:00', '--job-id', '815', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp_fzjp09o', '--error-file', '/tmp/tmpp2bye9lb']
[2022-06-27 13:33:17,192] {standard_task_runner.py:80} INFO - Job 815: Subtask model_A
[2022-06-27 13:33:17,402] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_A scheduled__2022-06-14T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:33:17,716] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_A
AIRFLOW_CTX_EXECUTION_DATE=2022-06-14T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-14T00:00:00+00:00
[2022-06-27 13:33:17,720] {python.py:173} INFO - Done. Returned value was: 4
[2022-06-27 13:33:17,776] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_A, execution_date=20220614T000000, start_date=20220627T133316, end_date=20220627T133317
[2022-06-27 13:33:17,880] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:33:17,986] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
